prep-html-links/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <div>
+    <a href="http://127.0.0.1:5500/prep-html-links/contact-us/contact-us.html">Click here to go to the contact page</a>
```

Comment:
Rather than use the absolute URL to the `contact-use.html` of your Live Server, use a relative path to the `contact-us.html` inside of the nearby `contact-us/` directory (_i.e._ `href="contact-us/contact-us.html"`).

That way, the link works on computers other than your own.